web/docs/language/features.md

```diff
@@ -178,138 +178,246 @@ Currently entities can be accessed only in Operations (Queries & Actions), so ch
 
 ## Queries and Actions (aka Operations)
 
-In Wasp, main interaction between client and server happens via Operations, of which two types exist: Queries and Actions.
+In Wasp, the client and the server interact with each other through Operations.
+Wasp currently supports two kinds of Operations: **Queries** and **Actions**.
 
 ### Query
 
-Queries are NodeJS functions that don't modify any state. Normally they fetch certain resources, process them and return result. They are executed on server. 
+Queries are resource-fetching functions that don't modify the state. Each Query should fetch, process, and return a particular resource.
```

Comment:
I realize the word "resource" was already in the old text, but that doesn't mean we shouldn't review it. Resource is a bit overloaded on the web. It may be easily confused with the notion of an HTTP resource. Why even use the word? What is a "particular resource"? The word seems rather fuzzy in its definition; we're really talking about query results and they're not particular at all but could have overlaps.

"Queries are used to fetch data from the server. Queries, unlike actions, do not modify the server's state."

Reply:
I agree, your suggestion is much clearer.

Reply:
nice, I like it!

Reply:
> "Queries are used to fetch data from the server. Queries, unlike actions, do not modify the server's state."

This might a bit too specific though -> they don't necessarily have to fetch data from the server, they could fetch data from some external service, or maybe generate data themselves. That said, they are executing on the server (for now), so it is in a way correct that the data is coming from the server, even if originating from somewhere else or constructed on the fly. Okkkkk